File: app/src/main/java/com/cookplan/recipe/import_recipe/search_url/SearchRecipeUrlPresenter.java
package com.cookplan.recipe.import_recipe.search_url;

/**
 * Created by DariaEfimova on 10.06.17.
 */

public interface SearchRecipeUrlPresenter {

    void searchRecipes(String query);

    void loadNextPart(int offset);

    void onStop();
}

File: app/src/main/java/com/cookplan/recipe/import_recipe/search_url/SearchRecipeUrlPresenter.kt
package com.cookplan.recipe.import_recipe.search_url

/**
 * Created by DariaEfimova on 10.06.17.
 */

interface SearchRecipeUrlPresenter {

    fun searchRecipes(query: String)

    fun loadNextPart(offset: Int)

    fun onStop()
}
